toolchain/emcc.sh

```diff
@@ -18,12 +18,25 @@ set -euo pipefail
 
 . $(dirname $0)/common.sh
 
-emcc -s EMIT_EMSCRIPTEN_METADATA=1 -s STANDALONE_WASM=1 -s EXPORTED_FUNCTIONS=['_malloc','_free'] "$@"
+# Remove coverage flags which are not supported.
+args=($@)
+delete=--coverage
```

Comment:
Nit: could you put those in quotes to improve readability? (Feel free to ignore this if you disagree)